src/config.ts

```diff
@@ -254,32 +261,47 @@ const EXCLUDED_SOURCES = (() => {
                 ERC20BridgeSource.Kyber,
                 ERC20BridgeSource.Uniswap,
                 ERC20BridgeSource.UniswapV2,
-                ERC20BridgeSource.UniswapV2Eth,
                 ERC20BridgeSource.MultiBridge,
+                ERC20BridgeSource.Balancer,
             ];
     }
 })();
 
-export const GAS_SCHEDULE: { [key in ERC20BridgeSource]: number } = {
-    [ERC20BridgeSource.Native]: 1.5e5,
-    [ERC20BridgeSource.Uniswap]: 3e5,
-    [ERC20BridgeSource.UniswapV2]: 3.5e5,
-    [ERC20BridgeSource.UniswapV2Eth]: 4e5,
-    [ERC20BridgeSource.LiquidityProvider]: 3e5,
-    [ERC20BridgeSource.MultiBridge]: 6.5e5,
-    [ERC20BridgeSource.Eth2Dai]: 5.5e5,
-    [ERC20BridgeSource.Kyber]: 8e5,
-    [ERC20BridgeSource.CurveUsdcDai]: 9e5,
-    [ERC20BridgeSource.CurveUsdcDaiUsdt]: 9e5,
-    [ERC20BridgeSource.CurveUsdcDaiUsdtTusd]: 10e5,
-    [ERC20BridgeSource.CurveUsdcDaiUsdtBusd]: 10e5,
-    [ERC20BridgeSource.CurveUsdcDaiUsdtSusd]: 6e5,
+export const GAS_SCHEDULE: FeeSchedule = {
+    [ERC20BridgeSource.Native]: () => 1.5e5,
+    [ERC20BridgeSource.Uniswap]: () => 3e5,
+    [ERC20BridgeSource.LiquidityProvider]: () => 3e5,
+    [ERC20BridgeSource.Eth2Dai]: () => 5.5e5,
+    [ERC20BridgeSource.Kyber]: () => 8e5,
+    [ERC20BridgeSource.Curve]: fillData => {
+        switch ((fillData as CurveFillData).poolAddress.toLowerCase()) {
+            case '0xa2b47e3d5c44877cca798226b7b8118f9bfb7a56':
```

Comment:
can we make these addresses constants somewhere? Maybe even just an obj in this file.

```
CurvePoolAddresses = {
    USD_ETH: '...',
}
```